src/viur/core/email.py

```diff
@@ -449,3 +458,41 @@ def check_sib_quota() -> None:
             entity["latest_warning_for"] = None
 
         db.Put(entity)
+
+
+if 'mailjet_rest' in sys.modules:
+    class EmailTransportMailjet(EmailTransport):
+        @staticmethod
+        def deliverEmail(*, sender: str, dests: list[str], cc: list[str], bcc: list[str], subject: str, body: str,
+                         headers: dict[str, str], attachments: list[dict[str, bytes]], **kwargs):
+            data = {"messages": [{}]}
+            email = data["messages"][0]
+            email["from"] = EmailTransportMailjet.splitAddress(sender)
+            email["to"] = [EmailTransportMailjet.splitAddress(dest) for dest in dests]
+            email["htmlpart"] = body
+            email["subject"] = subject
+            if bcc:
+                email["bcc"] = [EmailTransportMailjet.splitAddress(b) for b in bcc]
+            if cc:
+                email["cc"] = [EmailTransportMailjet.splitAddress(c) for c in cc]
+            if headers:
+                email["headers"] = headers
+            if attachments:
+                atts = []
+                for att in attachments:
+                    next_att = {
+                        "filename": att["filename"],
+                        "base64content": b64encode(att["content"]).decode("ASCII")
```

Comment:
`base64` was already imported above, therefore its not needed to extra-import `b64encode`.
```suggestion
                        "base64content": base64.b64encode(att["content"]).decode("ASCII"),
```